### DISCUSSION TITLE
Control Panel (user settings): Delete Trophies

### DISCUSSION THREAD

**Discussion by kdecks** [RetroAchievements/RAWeb#658] [Ideas]
![image](https://user-images.githubusercontent.com/32706333/36223817-d5bc52d4-1182-11e8-8d10-188fa099026b.png)
http://retroachievements.org/controlpanel.php

Right now users can reset any or all of their achievements but trophies remain and users may want for them to be removed as well.

Add the ability in a similar interface for players to permanently remove some or all of their profile page trophies.

**Comment by luchaos** [comment]
Note to self: move achievements and awards related actions to separate user settings page (done)

**Comment by ghost** [comment]
I assume 'Trophies' mean the Mastery Badges on the right in the Users Profile, if not then i will create a new Issue regards this.

Such function would be welcome, before v2 if possible.

**Comment by kdecks** [comment]
Yes it does, also #380
